packages/core-app-api/api-report.md

```diff
@@ -445,6 +445,8 @@ export class LocalStorageFeatureFlags implements FeatureFlagsApi {
   // (undocumented)
   getRegisteredFlags(): FeatureFlag[];
   // (undocumented)
+  getSortedFlags(): FeatureFlag[];
```

Comment:
Let's not add this to the core API, and instead just sort them in `UserSettingsFeatureFlags`